salt/modules/junos.py

```diff
@@ -40,7 +44,12 @@
     from jnpr.junos.utils.scp import SCP
     import jnpr.junos.utils
     import jnpr.junos.cfg
+    import jnpr.junos.op as tables_dir
+    from jnpr.junos.factory.factory_loader import FactoryLoader
+    from jnpr.junos.factory.optable import OpTable
+    from jnpr.junos.exception import ConnectClosedError
     import jxmlease
+    import yamlordereddictloader
```

Comment:
Is there a reason you can't use the functions/classes in `salt/utils/yaml*` instead?

Reply:
Yeah, we should be using salt/utils/yaml* for all this yaml stuff so that we make sure we use our defined safeloader.